go.sum

```diff
@@ -1,7 +1,7 @@
 github.com/corazawaf/coraza-wasilibs v0.1.0 h1:cYsblr346P5cA49g6YQwhS1ygUQg5egwAywwsAbsUIo=
 github.com/corazawaf/coraza-wasilibs v0.1.0/go.mod h1:aMVO6E4TFAxXnPmyrrEoXVYeMDovq3IsKwuetAR38JE=
-github.com/corazawaf/coraza/v3 v3.1.0 h1:CB6YxNXdbZjUJS/0FVFoFvS8eOVFbIvlNuHNC5dh88c=
```

Comment:
To be resolved before Merging: point to a CRS tagged version with https://github.com/corazawaf/coraza/pull/1027 in